### ISSUE TITLE
In dashboard.conf (Nginx) /media should serve contents inside /static

### ISSUE THREAD

**Issue by sevein** [artefactual-labs/ansible-role-archivematica-src#59]
I think that we should run `collectstatic` in the dashboard to put all the assets under `/static`.

This is basically the change I am suggesting:
```nginx
   location /media {
-    alias /usr/share/archivematica/dashboard/media;
+    alias /usr/share/archivematica/dashboard/static;
   }
```

The problem with making this change is that we'd need an extra `location` for development environments pointing to `/usr/share/archivematica/dashboard/media` - this method however has some disadvantages like not allowing to locate assets in more than one subdirectory (Django apps). Ideally, we would serve the static assets from Django when doing development but that would require a couple of changes in the code. This is the approach that I'd prefer but I'm not sure if it's doable to perform such change across all our active code branches.

Basically we would be making two changes:

1. Add `STATIC_URL = "/static/"` to `settings/local.py` and,
2. Add a set of extra urls to `urls.py` only when `settings.DEBUG = True`
